### PR TITLE
Setup CI for documentation release with Azure Pipelines

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
- add: Added new azure pipeline for building and publishing the creator documentation.

### CODE REVIEW

File: .azure-documentation-release.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+      - task: PublishPipelineArtifact@1
+        inputs:
+          targetPath: 'Creator/.Documentation/_site'
+          artifact: 'documentation'
+          publishLocation: 'pipeline'
+        displayName: Store Documentation
+
+      - task: FtpUpload@2
+        displayName: "Uploading to server"
```
Comment (LukasReindlGit) [inline .azure-documentation-release.yml:81]: Only step that could be considered "release" maybe it makes sence to also call it like that. Ideally move the release part to a separate job/stage

File: .azure-documentation-release.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+parameters:
+- name: release
+  type: boolean
+  default: false
+
+pool: Unity
+
+jobs:
+  - job:
```
Comment (LukasReindlGit) [inline .azure-documentation-release.yml:27]: name the job

Comment (tomwim) [reply .azure-documentation-release.yml:27]: named the job

**Comment by tomwim** [comment]
Please re-review @LukasReindlGit

**Comment by tomwim** [comment]
Please re-review @LukasReindlGit

**Pull request merged**